scm/driver/github/pr.go

```diff
@@ -154,7 +155,7 @@ func convertPullRequest(from *pr) *scm.PullRequest {
 		Link:   from.HTMLURL,
 		Diff:   from.DiffURL,
 		Closed: from.State != "open",
-		Merged: from.MergedAt.String != "",
```

Comment:
can you provide some more details as to why this does not work?  I cannot recall, but there was a reason we did not use `merged` (perhaps compatibility with older versions of github enterprise, but I honestly cannot recall)

note that MergedAt handles null values and will interpret a null value as an empty string, so I would would expect this existing logic to work.

Reply:
The MergedAt is also workable, but I personally think the Merged field is more explicit.

Reply:
Because it is explicitly stated on the github doc.

Reply:
are you certain this will not break older version of github enterprise?  we have to consider strict backward compatibility when we use this library.  If we were writing from scratch today we could use `merged`, but we have a number of github enterprise customers using this library and many of them are using older versions of github enterprise.

Reply:
ok, I will revert this change.